package-configuration/linux-cli/get-cloudify.py

```diff
@@ -181,23 +189,28 @@ def make_virtualenv(virtualenv_dir, python_path):
 
 
 def install_module(module, version=False, pre=False, virtualenv_path=False,
-                   wheelspath=False, upgrade=False):
+                   wheelspath=False, requirement_files=None, upgrade=False):
     """This will install a Python module.
 
     Can specify a specific version.
     Can specify a prerelease.
     Can specify a virtualenv to install in.
+    Can specify a list of paths or urls to requirement txt files.
     Can specify a local wheelspath to use for offline installation.
     Can request an upgrade.
     """
     lgr.info('Installing {0}...'.format(module))
-    pip_cmd = 'pip install {0}'.format(module)
+    pip_cmd = 'pip install'
```

Comment:
I wonder if it won't be easier to leave pip_cmd as list:

```
pip_cmd = ['pip', 'install']
...
pip_cmd.extend(['-r', req_file])
```

this will save all the formatting?